clang/test/CodeGenCXX/debug-info-range-for-var-names.cpp

```diff
@@ -16,14 +16,14 @@ void test() {
 }
 
 // CHECK: call void @llvm.dbg.declare(metadata ptr %__range1, metadata ![[RANGE1:[0-9]+]]
-// CHECK: call void @llvm.dbg.declare(metadata ptr {{.*}}, metadata ![[BEGIN1:[0-9]+]]
-// CHECK: call void @llvm.dbg.declare(metadata ptr {{.*}}, metadata ![[END1:[0-9]+]]
+// CHECK: call void @llvm.dbg.declare(metadata ptr {{[^,]*}}, metadata ![[BEGIN1:[0-9]+]]
```

Comment:
What's this pre-script fixup about?

Reply:
Removing an overly aggressive capture group, which causes problems when `BEGIN1` can match multiple things (either the variable or the location).